atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -214,6 +214,10 @@ private void runTwoNoopsOnTwoHostsAndThrowFromSecondRunOnFirstHost(Exception exc
                 .isInstanceOf(Exception.class);
     }
 
+    private CassandraServer getServerForIndex(int index) {
+        return CassandraServer.from(InetSocketAddress.createUnresolved(Integer.toString(index), index));
```

Comment:
I meant to make this (public/package) static, and then use in other places where you're doing the `"0", 0` thing - but certainly not blocking on this! Fine to test/merge as-is.